ingestion_tools/scripts/importers/identified_object.py

```diff
@@ -0,0 +1,118 @@
+from common.metadata import IdentifiedObjectMetadata
+from importers.base_importer import BaseFileImporter
+
+
+class IdentifiedObjectIdentifierHelper(IdentifierHelper):
+    @classmethod
+    def _get_container_key(cls, config: DepositionImportConfig, parents: dict[str, Any], *args, **kwargs) -> str:
+        return "-".join(["identified_object", parents["run"].get_output_path()])
+
+    @classmethod
+    def _get_metadata_glob(cls, config, parents, *args, **kwargs) -> str:
+        run = parents["run"]
+        metadata_glob = config.resolve_output_path("identified_object_metadata", run)
+        return metadata_glob
+
+    @classmethod
+    def _generate_hash_key(
+        cls,
+        container_key: str,
+        metadata: dict[str, Any],
+        parents: dict[str, Any],
+        *args,
+        **kwargs,
+    ) -> str:
+        return "-".join(
+            [
+                container_key,
+                str(metadata.get("deposition_id", int(parents["deposition"].name))),
+                str(metadata.get("run_id", parents["run"].name)),
+                str(metadata.get("identified_object_id", "unknown")),
+                str(metadata.get("identified_object_name", "unknown")),
+                str(metadata.get("identified_object_state", "unknown")),
+                str(metadata.get("identified_object_description", "unknown")),
+            ],
+        )
+
+
+class IdentifiedObjectImporter(BaseFileImporter):
+    type_key = "identified_object"
+    plural_key = "identified_objects"
+    finder_factory = DefaultImporterFactory
+    has_metadata = True
+    dir_path = "{dataset_name}/{run_name}/IdentifiedObjects"
+    metadata_path = "{dataset_name}/{run_name}/IdentifiedObjects/identified_objects_metadata.json"
+
+    def __init__(
+        self,
+        config: DepositionImportConfig,
+        metadata: dict[str, Any],
+        name: str,
+        path: str,
+        allow_imports: bool,
+        parents: dict[str, Any],
+    ):
+        super().__init__(
+            config=config,
+            metadata=metadata,
+            name=name,
+            path=path,
+            allow_imports=allow_imports,
+            parents=parents,
+        )
+        self.identifier = IdentifiedObjectIdentifierHelper.get_identifier(
+            config, self.get_base_metadata(), self.parents,
+        )
+
+    def import_item(self) -> None:
+        if not self.is_import_allowed():
+            print(f"Skipping import of {self.name}")
+            return
+
+        dest_path = self.get_output_path()
+        self.config.fs.makedirs(dest_path)
+        try:
+            with self.config.fs.open(self.path, "r") as f:
+                df = pd.read_csv(f)
+        except Exception as e:
+            print(f"Error reading CSV {self.path}: {e}")
+            return
+        output_file = f"{dest_path}/identified_objects.json"
+        with self.config.fs.open(output_file, "w") as f:
+            df.to_json(f, orient='records', indent=4)
```

Comment:
Sorry if I am misunderstanding the usecase. But, it looks like we would have a single csv file for a dataset, that also includes the run name as a column. 

If that is correct, 
1. Should there be a filter by the run name before we write the data? 
2. Should that column be excluded in the data written?

cc: @uermel